templates/organization/commissionable_assets_list.html.twig

```diff
@@ -0,0 +1,25 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Espace structure{% endblock %}
+
+{% block body %}
+    <h1>Véhicules</h1>
+
+    <p><a class="btn btn-link" href="{{ path('organization_home') }}" role="button">Retourner à l'accueil</a> {{ organization.name }}</p>
+
+    <ul class="list-group list-group-flush mb-4">
+        {% for asset in assets %}
+        <li class="list-group-item list-group-item-action">
+            <div class="d-flex justify-content-between align-items-center">
+                <span>{{ asset.type ~ ' - ' ~ asset.name }}</span>
```

Comment:
maybe make this the `__toString()` method implementation or at least a dedicated method to prevent complex computations in Twig